packages/vue/src/components/authenticator.vue

```diff
@@ -109,21 +92,22 @@ unsubscribeMachine = service.subscribe((newState) => {
   }
 }).unsubscribe;
 
+const { route, signOut, toSignIn, toSignUp, user } = toRefs(
+  // `useAuthenticator` is casted for temporary type safety on this file.
+  useAuthenticator() as AuthenticatorServiceFacade
```

Comment:
I added temporary type safety to this file so that we can trust at least the type safety of the changes I make below.